File: src/CS/XunitClassData/ClassData.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace $rootnamespace$
{
    public class $safeitemrootname$ : IEnumerable<object[]>
    {
        public IEnumerator<object[]> GetEnumerator()
        {
            yield return new object[] { new object() };
        }

        IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
    }
}

File: src/VB/XunitClassData/ClassData.vb
Public Class $safeitemname$
    Implements IEnumerable(Of Object())

    Public Iterator Function GetEnumerator() As IEnumerator(Of Object()) Implements IEnumerable(Of Object()).GetEnumerator

        Yield New Object() {New Object()}

    End Function

    Private Function GetUntypedEnumerator() As IEnumerator Implements IEnumerable.GetEnumerator
        Return GetEnumerator()
    End Function

End Class